grid-proxy/docs/database.md

```diff
@@ -0,0 +1,11 @@
+# Database
+
+The grid proxy has access to a postgres database containing information about the tfgrid.\
+The grid proxy mainly retrieves information from the db (maybe with a few modifications for efficient retrieval)\
+
+## Max Open Connections
+
+The db can handle 100 open connections concurrently (that is the default value for postgres).\
+The explorer creates a connection pool to the postgres db, with a max open pool connections set to a specific number (currently 80).\
```

Comment:
Why 80 was chosen? can i configure it?

Reply:
it is explained in the document why 80 is used [here](https://github.com/threefoldtech/tfgrid-sdk-go/pull/371/files#diff-daa7b575edcdbd3ca9d60f42426b2519378c4db7c4bb5bc31397e066f912cf2dR11)

it is because the max connections limit from postgres is 100, we could set it to 100, but then if there is one open connection from another actor, other than the explorer, the too many connections error may result in the explorer or the other actor, this is why it is set to 80.

Reply:
i'll add the configuration part